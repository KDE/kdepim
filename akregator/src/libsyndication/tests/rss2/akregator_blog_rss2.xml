<?xml version="1.0" encoding="utf-8"?>
<!-- generator="wordpress/1.5.1.2" -->
<rss version="2.0" 
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
>
<!--
     Source: your's truly
     properties: 
      - example for Wordpress 1.5 RSS2
       - description: CDATA plaintext with entities
       - content: CDATA html
       
-->
<channel>
	<title>akregator blog</title>
	<link>http://akregator.pwsp.net/blog</link>
	<description>akregator developer's blog</description>
	<pubDate>Thu, 15 Dec 2005 13:05:33 +0000</pubDate>
	<generator>http://wordpress.org/?v=1.5.1.2</generator>
	<language>en</language>

		<item>
		<title>Qt4/KDE4 build instructions</title>
		<link>http://akregator.pwsp.net/blog/?p=26</link>
		<comments>http://akregator.pwsp.net/blog/?p=26#comments</comments>
		<pubDate>Thu, 15 Dec 2005 13:05:30 +0000</pubDate>
		<dc:creator>frank</dc:creator>
		
	<category>General</category>
		<guid>http://akregator.pwsp.net/blog/?p=26</guid>
		<description><![CDATA[	I promised in my last entry to blog about how to build akregator from trunk.
Well, Thorsten Staerk and Allen Winter already did the work, so I just point you to the KDE3to4 wiki page.
	kdebase is optional, you can just skip it (as it doesn&#8217;t always build) and go for kdepim directly. See previous blog entry [...]]]></description>
			<content:encoded><![CDATA[	<p>I promised in my last entry to blog about how to build akregator from trunk.<br />
Well, Thorsten Staerk and Allen Winter already did the work, so I just point you to <a href="http://wiki.kde.org/tiki-index.php?page=KDE3To4">the KDE3to4 wiki page</a>.</p>
	<p>kdebase is optional, you can just skip it (as it doesn&#8217;t always build) and go for kdepim directly. See previous blog entry how to build and install akregator.<br />
You might also read README.qt-copy in qt-copy/ to learn how to speed up Qt4 compilation (i.e., don&#8217;t build examples and such).
</p>
]]></content:encoded>
			<wfw:commentRSS>http://akregator.pwsp.net/blog/?feed=rss2&amp;p=26</wfw:commentRSS>
	</item>
	</channel>
</rss>
