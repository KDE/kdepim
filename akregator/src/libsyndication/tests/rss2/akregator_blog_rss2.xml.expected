# Feed begin ######################
title: #akregator blog#
link: #http://akregator.pwsp.net/blog#
description: #akregator developer's blog#
language: #en#
# Item begin ######################
id: #http://akregator.pwsp.net/blog/?p=26#
title: #Qt4/KDE4 build instructions#
link: #http://akregator.pwsp.net/blog/?p=26#
description: #I promised in my last entry to blog about how to build akregator from trunk.<br/>Well, Thorsten Staerk and Allen Winter already did the work, so I just point you to the KDE3to4 wiki page.<br/> kdebase is optional, you can just skip it (as it doesnâ€™t always build) and go for kdepim directly. See previous blog entry [...]#
content: #<p>I promised in my last entry to blog about how to build akregator from trunk.<br /> Well, Thorsten Staerk and Allen Winter already did the work, so I just point you to <a href="http://wiki.kde.org/tiki-index.php?page=KDE3To4">the KDE3to4 wiki page</a>.</p> <p>kdebase is optional, you can just skip it (as it doesn&#8217;t always build) and go for kdepim directly. See previous blog entry how to build and install akregator.<br /> You might also read README.qt-copy in qt-copy/ to learn how to speed up Qt4 compilation (i.e., don&#8217;t build examples and such). </p>#
datePublished: #Thu Dec 15 14:05:30 2005#
dateUpdated: #Thu Dec 15 14:05:30 2005#
# Person begin ####################
name: #frank#
# Person end ######################
# Category begin ##################
term: #General#
# Category end ####################
commentsLink: #http://akregator.pwsp.net/blog/?p=26#comments#
commentsFeed: #http://akregator.pwsp.net/blog/?feed=rss2&p=26#
# Item end ########################
# Feed end ########################

